.github/workflows/deploy_lambda_dev.yml

```
@@ -0,0 +1,24 @@
+name: Deploy Lambdas to Dev
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: Branch to deploy
+        required: true
+        default: master
+
+jobs:
+  setup-job:
```

```suggestion
  setup_job:
```
I hate to do this but I was reviewing your other PR in infra and noticed they're underscores.

---

Changed to `_`.